tests/functional/change_password.js

```diff
@@ -308,7 +308,7 @@ define([
           return FunctionalHelpers.getVerificationLink(email, 0);
         })
         .then(function (verificationLink) {
-          return self.get('remote').get(require.toUrl(verificationLink));
+          return self.remote.get(require.toUrl(verificationLink));
```

Comment:
After seeing this 400 or so times, part of me feels like we should have abstracted this away. That was largely my intent with https://github.com/mozilla/fxa-content-server/blob/dc048832e1db52f46621177bf98a4f203714c204/tests/functional/lib/helpers.js#L587. Perhaps as part two, or a separate PR, we could replace most of the boilerplate with a call to the helper.